app/main.go

```diff
@@ -74,16 +75,25 @@ func main() {
 }
 
 func setup(o opts) (n notifier, p publisher.Interface, err error) {
+	content, err := ioutil.ReadFile("exclusion-patterns.txt")
+	if err != nil {
+		log.Print("[WARN] could not read 'exclusion-patterns.txt' file")
```

Comment:
keeping context is important, but showing the actual error is even more important, `i.e. log.Printf("[WARN] could not read 'exclusion-patterns.txt' file, %v", err)`

Reply:
Explicit sad path return added

Reply:
Added logging of the error to the error message